SECURITY.md

```diff
@@ -1,34 +1,93 @@
 # Security policy
 
-## Supported versions
+## Warning to users
+
+PowSyBl is a library dedicated to electrical grid modelling and simulation. As such, it includes data importers to feed the calculations. Please make sure your data comes from a safe source.
 
-The available compatible releases are listed in [powsybl-dependencies](https://github.com/powsybl/powsybl-dependencies). In case of vulnerability fix, the main branches are first fixed. The fix will be available for the next release train. In case of vulnerability of high or critical severity, the most recent version is patched.
+PowSyBl also provides a persistent IIDM implementation by offering the possibility to connect to an external PostgreSQL database. If you use this feature, please make sure your login and password are secure and safely stored.
```

Comment:
We also have "AFS" that is used to organize studies.

Reply:
Yes. Finally, I have not included details about repositories but kept it general. Does it suit you?